test/MUXControlsTestApp/verification/NavigationViewAuto-4.xml

```diff
@@ -2517,3 +2517,18 @@
                 FocusVisualPrimaryBrush=#FF000000
                 Visibility=Collapsed
                 RenderSize=0,0
+      [Windows.UI.Xaml.Controls.Grid]
```

Comment:
Not related to this PR, but why are those .xml files if the content is not XML? @kaiguo @StephenLPeters do you know why we went for xml as file extension?